packages/knip/test/cli-reporter-junit.test.ts

```diff
@@ -0,0 +1,36 @@
+import assert from 'node:assert/strict';
+import test from 'node:test';
+import { resolve } from '../src/util/path.js';
+import { execFactory } from './helpers/exec.js';
+
+const cwd = resolve('fixtures/module-resolution-non-std');
+
+const exec = execFactory(cwd);
+
+test('knip --reporter junit', () => {
+  const xml = `
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites name="Knip report" tests="4" failures="4">
+  <testsuite name="Unused files" tests="1" failures="1">
+    <testcase tests="1" failures="1" name="Unused files" classname="src/unused.ts">
+      <failure message="Unused files" type="Unused files">Unused files: src/unused.ts</failure>
+    </testcase>
+  </testsuite>
+  <testsuite name="Unlisted dependencies" tests="2" failures="2">
+    <testcase tests="1" failures="1" name="Unlisted dependencies" classname="src/index.ts">
+      <failure message="Unlisted dependencies - unresolved" type="Unlisted dependencies">Unlisted dependencies: "unresolved" inside src/index.ts</failure>
```

Comment:
I don't know anything about this format, but will each issue be in its own node, or should it be? This example gives me the impression that all unlisted dependencies will end up in this single node? Or will each one have the "Unlisted dependencies: " prefix?

Maybe you could show an example of how it looks like in the CI environment (eg. Bitbucket)?